python/res/job_queue/job_queue_node.py

```diff
@@ -71,33 +75,40 @@ def is_running(self):
                 self.status == JobStatusType.JOB_QUEUE_RUNNING  or
                 self.status == JobStatusType.JOB_QUEUE_UNKNOWN) # dont stop monitoring if LSF commands are unavailable
     
-    def job_monitor(self, driver):
+    def job_monitor(self, driver, max_submit):
```

Comment:
Should we make this private, i.e. `_job_monitor`?

Reply:
sure